cfgov/paying_for_college/documents.py

```diff
@@ -20,6 +23,10 @@ def prepare_autocomplete(self, instance):
     def prepare_nicknames(self, instance):
         return [n.nickname for n in instance.nickname_set.all()]
 
+    def prepare_url(self, instance):
+        return reverse("paying_for_college:disclosures:school-json",
+                       args=[instance.school_id])
+
```

Comment:
We need to exclude schools that have closed from being indexed. The ES7 index is finding 8,693 schools, but it should only find 6,910, because the others have closed.

We can override the document's `get_queryset` method to accomplish this:

```suggestion

    def get_queryset(self):
        """Prevent schools that have closed from being indexed."""
        return super(
            SchoolDocument, self).get_queryset().filter(operating=True)

```

Reply:
That explains why the number of schools found in the index was incorrect.
I made the change necessary to filter on schools that are operating which should exclude schools that are closed.